src/main/java/com/linkedin/replica/wall/commands/impl/EditReplyCommand.java

```diff
@@ -53,7 +52,7 @@ public Object execute() throws ParseException {
         reply.setImages(images);
         reply.setLikesCount(likesCount);
         reply.setUrls(urls);
-        reply.setTimestamp(timestamp);
+        reply.setTimestamp(reply.getTimestamp());
         reply.setText(text);
         String response = dbHandler.editReply(reply);
```

Comment:
Edit using AQL edit

Reply:
I think we are editing too much attributes why using AQL while we can editing the object using the setters and getters. What is the purpose of using models then ?

Reply:
asically, it's a bad practice to replace a whole item each time a minor change occur